data/styles/clear/include/Basemap.mapcss

```diff
@@ -704,13 +704,10 @@ line|z16-[barrier=wall],
 {color: @fence; width: 1; opacity: 0.7;}
 line|z17[barrier=fence],
 line|z17[barrier=wall],
-{width: 1.3; opacity:0.8;}
-line|z18[barrier=fence],
-line|z18[barrier=wall],
-{width: 2; opacity: 1;}
-line|z19-[barrier=fence],
-line|z19-[barrier=wall],
-{width: 3; opacity: 1;}
+{width: 1.2; opacity: 0.8;}
+line|z18-[barrier=fence],
+line|z18-[barrier=wall],
+{width: 1.3; opacity: 1;}
```

Comment:
AFAIR, we made fences _bolder_ intentionally some time ago. And I'm used to it :)
It looks fine now on my iPhoneX.
Keep 1.3 for z17 and 1.5 for z18- ?

Reply:
AFAIR one of the reasons to make them bold was to avoid confusion with power lines. But now power lines are distinguished by the pattern...
Re specific numbers - its depends on a particular device, e.g. in my emulator going from 1.3 to 1.4 is a visible boost already...

Reply:
We also had some claims in support to make fences more visible before ...